### ISSUE TITLE
Generate simulated training and test dataset

### ISSUE THREAD

**Issue by rcurrie** [braingeneers/organoid-net#3]
(moving from https://github.com/braingeneers/braingeneers/issues/3)

Generate simulated data with inputs and outputs as a proxy for future MEA data. Training data should provide labels as to which outputs are connected to inputs. Test data should not. Nominal idea is to wrangle this into PRP CEPH and run DeepCorr against it to see if it can learn the correlations and predict the test sets input/output pairs.

**Comment by masonhargrave** [comment]
[Hargrave_Organoid_Model.pdf](https://github.com/braingeneers/organoid-net/files/2597099/Hargrave_Organoid_Model.pdf)
Here  is a brief explanation of my first model.

**Comment by atspaeth** [comment]
Hey, can you actually prove your eqn (1)? The fact that it doesn't work for the S=(0,0) case is really suspicious, but I'm not sure what the issue is yet.

**Comment by masonhargrave** [comment]
It works for every case other than $\mathbf{S_t}=\mathbf{0}$.

**Comment by masonhargrave** [comment]
The final matrix formulation doesn't actually work. This model will make significantly more sense when implemented in a Python notebook. I should have that up and running soon.

**Comment by masonhargrave** [comment]
Here is an updated model with a working equation (1). It still doesn't work for the S=(0,0) case.

**Comment by atspaeth** [comment]
Forgot attachment?

**Comment by masonhargrave** [comment]
Oops yes, and turns out this totally works for all cases. Crisis averted!
[Hargrave_Organoid_Model (4).pdf](https://github.com/braingeneers/organoid-net/files/2597697/Hargrave_Organoid_Model.4.pdf)

**Comment by atspaeth** [comment]
That's more like it! And I'm willing to believe that this one does generalize to higher dimensions since now the cross terms are explicitly included. Can you forward-propagate in time easily, i.e. find E[S_{t+k}], by applying the same transformation k times? Perhaps if you replace each p_{i,j} in your product with p_{i,j} s_j ?

**Comment by masonhargrave** [comment]
I like the idea of replacing p_{i,j} with p_{i,j}s_j as this removes the need for the set F_t. Asforthe forward propagation, I am yet unsure. I should have a jupyter notebook to push by tonight. Let's discuss tomorrow after class!

**Comment by masonhargrave** [comment]
Note sure how to push code yet (someone will have to teach me) but the first version of my model is complete.
